src/App.js

```diff
@@ -1,13 +1,28 @@
 import React, { Component } from 'react'
-import { Instructions } from './components'
+import { Instructions, Counter, InitialCountForm } from './components'
 import './styles/app.css'
 
 class App extends Component {
+  state = {countSubmitted: false, invalidInput: false}
+  displayCounter = ev => {
+    ev.preventDefault()
+    let input = ev.target[0].value
```

Comment:
use `const` unless you need to reassign variables

https://github.com/airbnb/javascript#references--prefer-const